src/test/scala/okapies/finagle/kafka/ClientTest.scala

```diff
@@ -41,6 +43,10 @@ trait KafkaTest extends BeforeAndAfterAll { suite: Suite =>
     kafkaConfig = TestUtils.createBrokerConfig(1)
     kafkaConfig.put("zookeeper.connect", zkConn)
     kafkaConfig.put("host.name", "127.0.0.1")
+
+    // https://github.com/apache/kafka/blob/0.8.2/core/src/test/scala/unit/
+    // kafka/server/OffsetCommitTest.scala#L49
+    kafkaConfig.put("offsets.topic.replication.factor", "1")
```

Comment:
I think this workaround for [KAFKA-1867](https://issues.apache.org/jira/browse/KAFKA-1867) is no problem. (It seems will be fixed in 0.8.3)